source/class/qx/tool/compiler/Analyser.js

```diff
@@ -1415,6 +1419,47 @@ qx.Class.define("qx.tool.compiler.Analyser", {
       this.__librariesByNamespace[library.getNamespace()] = library;
     },
 
+    /**
+     * Returns a font by name
+     *
+     * @param {String} name
+     * @param {Boolean?} create whether to create the font if it does not exist (default is false)
+     * @returns {qx.tool.app.ManifestFont?} null if it does not exist and `create` is falsey
```

Comment:
```suggestion
     * @returns {qx.tool.app.ManifestFont?} null if it does not exist and `create` is false
```

Reply:
it's falsey not false, so the comment is correct